src/components/PersonalInfoForm/PersonalInfoForm.css

```diff
@@ -1,9 +1,8 @@
 .personal-form-container {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  margin-bottom: 2rem;
-  padding: 1rem;
+  width: 300px; /* Adjust the width as needed */
+  margin: 0 auto;
+  padding: 20px;
+  border-radius: 5px;
```

Comment:
It would be great to use relative units such as em and rem instead of pixel, in order to better support multiple screen sizes and resolutions.

Reply:
@fissoreg  Resolved